docs/kb/customizing-cpuset.rst

```diff
@@ -0,0 +1,105 @@
+
+==========================
+Customizing CPUSET
+==========================
+
+CPUSET in the ``/etc/scylla.d/cpuset.conf`` file is automatically configured by running the ``scylla_setup`` 
+(or ``scylla_cpuset_setup``) script. If you want to customize CPUSET, you must modify both the ``cpuset.conf`` 
+and ``perftune.yaml`` files.
+
+
+Note that:
+
+* The ``scylla-server`` service will try to generate ``/etc/scylla.d/perftune.yaml`` *only if* the file doesn't exist.
+* The ``scylla-server`` service will successfully generate ``/etc/scylla.d/perftune.yaml`` *only if* the value set in ``cpuset.conf`` 
```

Comment:
This statement is incorrect since 

```
commit 80917a1054f305d08a60107d868e86757ca3ee69
Author: Vlad Zolotarov <vladz@scylladb.com>
Date:   Wed Aug 24 20:03:36 2022 -0400

    scylla_prepare: stop generating 'mode' value in perftune.yaml
```